lib/cached_network_image.dart

```diff
@@ -71,6 +71,10 @@ class _CachedNetworkImageState extends State<CachedNetworkImage> {
 }
 
 class CacheManager {
+  static Duration inbetweenCleans = new Duration(days: 7);
+  static Duration maxAgeCacheObject = new Duration(days: 30);
+  static int maxNrOfCacheObjects = 200;
```

Comment:
How would developers be able to change these config parameters . The `inbetweenCleans` for example might be too big?
Same with `maxNrOfCacheObjects`.  if objects are small I might opt in to store more..?

Reply:
These are public statics, so they can be set when you start the app.